core/src/test/java/tech/pegasys/eth2signer/core/signing/FcBlsArtifactSignerTest.java

```diff
@@ -43,4 +45,17 @@ void identifierReturnsBlsAddress() {
         "t3sjhgtrk5fdio52k5lzanh7yy4mj4rqbiowd6odddzprrxejgbjbl2irr3gmpbf7epigf45oy7asljj3v3lva";
     assertThat(identifier).isEqualTo(expectedBlsAddress);
   }
+
+  @ParameterizedTest
+  @CsvSource({
+    "'',qqsXe9+kDAGOh8uKmI6+oGN90ydJ0jRELvxvf2VwAd+g3WRKT/bi65RlhiGTrzzCFP4gY0BiAMe3+ghInVFLs1CXBcYGol+RiPoFUew7UZ6ON1zq5Tjhs5F2vWFj1qt8",
+    "NDI=,p/LeJS8KPzuOco2qpqjhvJxFrA5qj++LpeVoActWkDRQTLhuGsnwAvY1kluQ6PntAqISjvR/RU0kzgkR38F8Hm5NuNPsu9/BEiU+IQheNeD7OoG8THq4OuZMbISg7uR/"
+  })
+  void signsData(final String message, final String expectedSignature) {
```

Comment:
any better name for the test? messagesComputedSignatureMatchesReferenceSignature